build/tasks.js

```diff
@@ -30,45 +30,45 @@ export default {
   },
 
   async run(args = []) {
-    await this.build();
+    await this.build([...args, '--force']);
```

Comment:
Maybe re-purpose that issue to follow this, or close that and file a new one instead?